src/js/tour.js

```diff
@@ -159,6 +161,8 @@ export class Tour extends Evented {
     cleanupSteps(this.tourObject);
     cleanupModal.call(this);
 
+    window.removeEventListener('touchmove', preventModalBodyTouch, false);
```

Comment:
Could this be moved into [`cleanupStepEventListeners`](https://github.com/shipshapecode/shepherd/pull/302/files#diff-78c335d6579502a5ab234e509a14fba0R160)?